studio/components/interfaces/Billing/PlanSelection/Plans/PlanCard.tsx

```diff
@@ -92,15 +54,9 @@ const PlanCard: FC<Props> = ({ plan, currentPlan, onSelectPlan }) => {
             )}
           </div>
         </div>
-        <div className={`${isEnterprise ? 'lg:space-y-0 lg:!mt-0 xl:space-y-4' : 'space-y-4'}`}>
+        <div className="space-y-4">
```

Comment:
for anyone reading this.. tailwind seems to default more to "flex flex-col gap-4" now for this type of stuff.

feels easier. but i suppose that is a styleguide thing we can just agree on later.